src/lib/composables/use-align.ts

```diff
@@ -34,7 +34,7 @@ const alignMap = {
   stretch: "stretch",
 } as const;
 
-export default function useAlign(align: UseAlignOptions = "top left") {
+export function useAlign(align: UseAlignOptions = "top left") {
```

Comment:
Why?

Reply:
Just for consistency, because other composables also are named exports